test/api/unit/services/SQS.test.js

```diff
@@ -80,8 +80,6 @@ describe('SQS', () => {
     context("building a site's default branch", () => {
       it('should set an empty string for BASEURL in the message for a site with a custom domain', (done) => {
         const domains = [
-          'www.example.com',
```

Comment:
Yes, but it still affects the value of `BASEURL` in the build settings. Changing that in the db could cause the `BASEURL` value to be incorrect for some live sites.